### ISSUE TITLE
[REQUEST] Implementing panel-by-panel navigation (Cocoapods library DeepPanel)

### ISSUE THREAD

**Issue by frykauf** [Paperback-iOS/app#477]
**Description**
Panels.app developers open-sourced their panel-by-panel navigation tech. 

It works by recognizing the individual panels in manga, and instead of page-by-page navigation, it reads panel-by-panel.

**How would it improve the App?**

Reading manga (unlike manhwa) can be a chore on small iphone display.
It’s just not large enough to display A5 pages correctly, this just leads to very subpar reading experience, where users easily miss the details or have hard time even following the manga while reading - since it’s too small.

This tech: pedrovgs/DeepPanel wants to improve it. By separating the reading into seperate panels. Users read by switching from panel to panel extraced from pages, instead of "turning pages“.

https://github.com/pedrovgs/DeepPanel 

This can also be combined with auto-rotating panels that are too wide (to show in widescreen) and too tall in portrait, ala "Smart Comic reader" app by Onne van Dijk. (slightly different panel-by-panel navigation, it uses zoom - less good).

**Additional context**
They have also premade it as Cocoapods library: 

https://github.com/pedrovgs/DeepPaneliOS

**Comment by aldoblack** [comment]
I've always wanted to open an issue regarding this. This is is great. [Panels - Comic Reader](https://apps.apple.com/us/app/panels-comic-reader/id1236567663) has implemented this, but it's not there yet. It needs a lot of work. 

I also suggest manual implementation. Like, you have all the panel coordinate in an object and the app just makes the transition.

**Comment by frykauf** [comment]
Yeah, in my opinion the Smart Comic reader’s implementation works better.
And something like a semi manual mode 

(like that it would zoom to a panel you tap on if you tap directly on it, instead of the back/forward navigation buttons)

would work the best IMO.

**Comment by frykauf** [comment]
But honestly if Paperback allowed to use external viewers (just acting as downloader) or had a real folder in storage we could just use whatever.

**Comment by celarye** [comment]
As part of a repository-wide cleanup for the 0.9 release, we are resetting our issue tracker to more efficiently track new feedback and feature requests. If this issue is still relevant or critical to your workflow, please feel free to reopen it with updated context. Thank you for your support!
